spec/features/admin/facilities/index_spec.rb

```diff
@@ -15,10 +15,11 @@
 
   let!(:protocol_01) { create(:protocol, name: "testProtocol") }
 
-  facility_page = AdminPage::Facilities::Show.new
+  facility_page = AdminPage::Facilities::Index.new
   facility_group = AdminPage::FacilityGroups::New.new
+  navigation = Navigations::DashboardPageNavigation.new
 
-  before(:each) do
+  before() do
```

Comment:
Let's remove the parentheses. It's [very common to do so](https://github.com/airbnb/ruby#no-args-parens) in Ruby.